dpgen/generator/run.py

```diff
@@ -1227,6 +1227,9 @@ def run_model_devi (iter_index,
 
     cwd = os.getcwd()
 
+    user_forward_files = mdata.get("model_devi" + "_user_forward_files", [])
+    forward_files += [os.path.basename(file) for file in user_forward_files]
+    backward_files += mdata.get("model_devi" + "_user_backward_files", [])
```

Comment:
the convention of xxx_user_forward_files, where xxx being model_devi, train, fp, is not explained in README.

Reply:
The directories of forward_common_files and forward_files are different. 
Take an example. If you ls 
```
JOBDIR/forward_common_file
JOBDIR/task.000.000/forward_file
JOBDIR/task.000.000/forward_file
```

Forward_common_files will be not put into the directories of tasks, this will cause the inconvenience for users. 
Actually we only  forward_common_files in two places: the model paths in training data set https://github.com/deepmodeling/dpgen/blob/da494650314b3eebcf31ff20a7c195b309cb1a6a/dpgen/generator/run.py#L573 and in model_devi https://github.com/deepmodeling/dpgen/blob/da494650314b3eebcf31ff20a7c195b309cb1a6a/dpgen/generator/run.py#L1195 . Even in processing POTCAR , we treated it as forward_files. https://github.com/deepmodeling/dpgen/blob/da494650314b3eebcf31ff20a7c195b309cb1a6a/dpgen/generator/run.py#L2163
 So that's why I create a new function `symlink_user_forward_files`

Reply:
OK I see the point